cmd/server-main.go

```diff
@@ -129,9 +129,8 @@ func serverHandleCmdArgs(ctx *cli.Context) {
 		fatalIf(checkPortAvailability(globalMinioPort), "Port %d already in use", globalMinioPort)
 	}
 
-	globalIsXL = (setupType == XLSetupType)
 	globalIsDistXL = (setupType == DistXLSetupType)
-	if globalIsDistXL {
+	if setupType == XLSetupType || globalIsDistXL {
```

Comment:
unnecessary change in this PR @donatello can you revert?

Reply:
Done.